bytes/src/main/scala/io/iohk/ethereum/utils/ByteUtils.scala

```diff
@@ -97,8 +97,8 @@ object ByteUtils {
 
   def padLeft(bytes: ByteString, length: Int, byte: Byte = 0): ByteString = {
     val l = math.max(0, length - bytes.length)
-    val fill = Seq.fill[Byte](l)(byte)
-    fill ++: bytes
+    val fill = Array.fill[Byte](l)(byte)
```

Comment:
Is this related to Scala 2.13 migration?

Reply:
Yes, `ByteString` can be concatenated only with `ByteString`
The result of `++: Seq` operation is an `IndexedSeq` in 2.13